Gemfile

```diff
@@ -1,46 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
Any particular reason you're removing the gemfile? I there's a decent number of things that aren't in chefdk listed here.